File: 2269-count-elements-with-strictly-smaller-and-greater-elements/count-elements-with-strictly-smaller-and-greater-elements.java
class Solution {
    public int countElements(int[] nums) {
        int count=0;
        int mn=Arrays.stream(nums).min().getAsInt();
        int mx=Arrays.stream(nums).max().getAsInt();
        for(int i:nums){
            if(mn<i && mx>i){
                count++;
            }
        }
        return count;
    }
}

File: 2269-count-elements-with-strictly-smaller-and-greater-elements/count-elements-with-strictly-smaller-and-greater-elements.py
class Solution(object):
    def countElements(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count=0
        mn=min(nums)
        mx=max(nums)
        for i in nums:
            if mn<i and mx>i:
                count+=1
            else:
                continue
        return count